lib/phoenix/channel/server.ex

```diff
@@ -147,9 +147,9 @@ defmodule Phoenix.Channel.Server do
   def reply(pid, ref, topic, {status, payload}, serializer)
       when is_binary(topic) and is_map(payload) do
 
-    send pid, serializer.encode!(
+    send pid, {:channel_push, serializer.encode!(
```

Comment:
Do you know why we even tag in the first place? I think it is because the driver process itself may receive Broadcast messages, right?